CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # bedrock-ledger-node ChangeLog
 
+## 16.0.0 - TBD
+
+### Changed
+- **BREAKING**: Upgraded `jsonld` to `^5.2.0` this might cause the `canonizedBytes`
+  & `hash` of records to change.
```

Comment:
Could use a reword as is, but also I think readers won't know what to do with that statement.  Perhaps a hybrid with the text I had proposed could help.  I tried to not delve into the details and suggest we don't think there are issues.
```suggestion
- **BREAKING**: Upgraded `jsonld` to `^5.2.0`. Marked as a breaking major change to help avoid any unknown upgrading issues in live systems due to potential JSON-LD edge case processing changes. No issues are expected in normal use cases. If such an issue exists, it might cause the `canonizedBytes` and `hash` of records to change.
```